content/en/user-guide/aws/ecs/index.md

```diff
@@ -338,6 +341,63 @@ services:
 
 Alternatively, you can download the image from the private registry before using it or employ an [Initialization Hook](https://docs.localstack.cloud/references/init-hooks/) to install the Docker client and use these credentials to download the image.
 
+
+## Running ECS on Kubernetes
+
+LocalStack Enterprise image allows you to run ECS tasks on Kubernetes. The tasks are added to ELB load balancer target groups. You can do so by setting the `ECS_TASK_EXECUTOR` environment variable to `kubernetes` in the LocalStack container.
+
+In this guide, you will learn how to run ECS tasks on Kubernetes by using [`k3d](https://k3d.io/), a lightweight Kubernetes distribution.
+
+### Create a new cluster
+
+After installing `k3d`, you can run the following commands to create a Kubernetes cluster:
+
+{{< command >}}
+$ export NODE_PORT=31566
+$ k3d cluster create ls-cluster -p "4566:$NODE_PORT" --wait --timeout 5m
+{{< / command >}}
+
+### Import LocalStack image
+
+You can import the LocalStack image to the Kubernetes cluster by running the following command:
```

Comment:
This is not needed for most people. We only do it in the pipeline to make sure we don't pull the public image